tox.ini

```
@@ -48,5 +48,22 @@ deps =
     pep517
     twine
 commands =
+    python -c "from pathlib import Path; \
+               [x.unlink(missing_ok=True) for x in Path('{toxinidir}/dist').glob('*')]"
     python -m pep517.build -s -b {toxinidir} -o {toxinidir}/dist
     twine check {toxinidir}/dist/*
+
+[testenv:release]
+description = Make a release; must be called after "build"
+skip_install = True
+deps =
+    twine
+depends =
+    build
+setenv =
+    TWINE_USERNAME={env:TWINE_USERNAME:INVALID USERNAME}
```

I'd probably just default to using a token
```suggestion
    TWINE_USERNAME={env:TWINE_USERNAME:__token__}
```

---

Yeah, another option here is to move these over to `passenv` and add `--non-interactive` into the default arguments.